src/style.css

```diff
@@ -0,0 +1,118 @@
+/* #region normalize*/
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 11px;
+  font-weight: 400;
+}
+
+body,
+p {
+  margin: 0;
+}
+/* #endregion*/
+
+/* #region card*/
+.card {
+  width: 200px;
+  padding: 14px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__picture {
+  margin: 16px 5px 45px ;
+}
+
+.picture__product-picture {
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+.card__product-name {
+  margin-bottom: 5px;
+}
+
+.product-name__name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+}
+
+.card__product-code {
+  margin-bottom: 15px;
+}
+/* #endregion*/
+
+/* #region rating*/
+.rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.rating-stars {
+  display: flex;
+}
+
+.rating-stars__star {
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  /*background-size: none;*/
+  background-position: center;
+}
+
+.rating-stars__star--active {
+  background-image: url(images/star-active.svg);
+}
+/* #endregion */
+
+/* #region price*/
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.price__value {
+  font-size: 16px;
+  font-weight: 700;
+}
+
+/* #endregion */
+
+/* #region button*/
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
+  height: 40px;
+
+  font-size: 14px;
+  font-weight: 700;
+
+  background-color: #00acdc;
+  border-radius: 5px;
+
+  color: #fff;
+  text-decoration: none;
+}
+
+.card__button:hover {
+  color: #00acdc;
+  background-color: #fff;
+  border: 1px solid #00acdc;
```

Comment:
at this moment when you hovered on your link - the border appears and adds 1px, because of box-sizing: border-box; width and height of your button is recalculating, and text could change its position, so it's better if you write border property inside the button.